test/browser/weighted.test.jsx

```diff
@@ -1,102 +1,83 @@
-import React from "react";
-import ReactDOM from "react-dom";
-import createReactClass from 'create-react-class';
-import Experiment from "../../src/Experiment.jsx";
-import Variant from "../../src/Variant.jsx";
-import emitter from "../../src/emitter.jsx";
-import assert from "assert";
-import co from "co";
-import UUID from "node-uuid";
-import TestUtils from 'react-dom/test-utils';
-import ES6Promise from 'es6-promise';
-ES6Promise.polyfill();
-
-let store;
-
-let noopStore = {
-  getItem: function(){},
-  setItem: function(){},
-  clear: function(){}
-};
-
-if(typeof window !== 'undefined' && 'localStorage' in window && window['localStorage'] !== null) {
-  try {
-    let key = '__pushtell_react__';
-    window.localStorage.setItem(key, key);
-    if (window.localStorage.getItem(key) !== key) {
-      store = noopStore;
-    } else {
-      window.localStorage.removeItem(key);
-      store = window.localStorage;
-    }
-  } catch(e) {
-    store = noopStore;
-  }
-} else {
-  store = noopStore;
-}
+import React from 'react';
+import UUID from 'uuid/v4';
+import { mount } from 'enzyme';
+
+import Experiment from '../../src/Experiment.jsx';
+import Variant from '../../src/Variant.jsx';
+import emitter from '../../src/emitter.jsx';
 
 function add(a, b) {
   return a + b;
 }
 
-describe("Weighted Experiment", function() {
-  this.timeout(10000);
-  let container;
-  before(function(){
-    container = document.createElement("div");
-    container.id = "react";
-    document.getElementsByTagName('body')[0].appendChild(container);
-  });
-  after(function(){
-    document.getElementsByTagName('body')[0].removeChild(container);
-    emitter._reset();
-  });
-  it("should choose a weighted variants.", co.wrap(function *(){
-    const experimentName = UUID.v4();
+describe('Weighted Experiment', () => {
+  xit('should choose a weighted variants', () => {
+    // TODO: this test work, but it's incredibly slow
```

Comment:
This test is currently skipped, because it takes ages to run, as it need to render the experiment 1000 times to measure the standard deviation.
Perhaps there's a better way of doing that. 🤔